src/darwin/Framework/CHIP/MTRBaseDevice.mm

```diff
@@ -2525,6 +2525,13 @@ - (instancetype)initWithPath:(const ConcreteCommandPath &)path
     return self;
 }
 
+- (NSString *)description
+{
+    return
+        [NSString stringWithFormat:@"<MTRCommandPath> endpoint %u cluster %lu command %lu", self.endpoint.unsignedShortValue,
```

Comment:
```suggestion
        [NSString stringWithFormat:@"<MTRCommandPath endpoint %u cluster %lu command %lu>", self.endpoint.unsignedShortValue,
```
Conventionally, the attributes go inside the `<>`

Reply:
OK, but should I then fix our other path classes to do that too?  Right now they don't @ksperling-apple .

Reply:
Answer is: yes, we should.  Doing that now.